slycot/CMakeLists.txt

```diff
@@ -131,21 +131,29 @@ set(CMAKE_Fortran_FLAGS )
 
 add_custom_target(wrapper ALL DEPENDS ${FSOURCES})
 add_custom_command(
-  OUTPUT SLYCOTmodule.c _wrappermodule.c _wrapper-f2pywrappers.f
+  OUTPUT _wrappermodule.c _wrapper-f2pywrappers.f
   COMMAND ${F2PY_EXECUTABLE} -m SLYCOT
   ${CMAKE_CURRENT_SOURCE_DIR}/${F2PYSOURCE}
   DEPENDS ${F2PYSOURCE_DEPS} ${F2PYSOURCE}
 )
 
 add_library(
   ${SLYCOT_MODULE} MODULE
-  SLYCOTmodule.c _wrappermodule.c _wrapper-f2pywrappers.f
-  "${PYTHON_SITE}/numpy/f2py/src/fortranobject.c"
+  _wrappermodule.c 
+  ${PYTHON_SITE}/numpy/f2py/src/fortranobject.c
```

Comment:
Just to check: CMake and scikit-build don't find this file for us, or arrange for an "f2py" target that takes care of this?